llarp/dht/node.hpp

```diff
@@ -46,6 +52,24 @@ namespace llarp
         introset.A.CalculateAddress(ID.as_array());
       }
 
+      void
+      ExtractStatus(util::StatusObject& obj) const override
+      {
+        obj.PutInt("timestamp", introset.T);
+
+        std::vector< util::StatusObject > introsObjs(introset.I.size());
+        size_t idx = 0;
+        for(const auto intro : introset.I)
+        {
+          auto& introObj = introsObjs[idx++];
```

Comment:
this is a little suspect just imo - I'm not a fan of putting `i++` inside longer sequences, because it makes it easy to introduce UB. transform gives something cleaner:

```c++
std::transform(introset.I.begin(), introset.I.end(), std::back_inserter(introsObjs), [](const auto& intro) -> util::StatusObject {
  util::StatusObject introObj;
  introObj.PutString("router", intro.router.ToHex());
  introObj.PutInt("expiresAt", intro.expiresAt);
  introObj.PutInt("latency", intro.latency);
  introObj.PutInt("version", intro.version);

  return introObj;
});
```

Reply:
Note combined with https://github.com/loki-project/loki-network/pull/300#discussion_r255365608, you might end up with:

```c++
util::StatusObject introObj{
  {"router", intro.router.ToHex()}, 
  {"expiresAt", intro.expiresAt},
  {"latency", intro.latency},
  {"version", intro.version}
};

return introObj;
```